### ISSUE TITLE
Pass both train and validation set to finetune() ?

### ISSUE THREAD

**Issue by mikerossgithub** [IndicoDataSolutions/finetune#405]
Is there a way to pass specific train/val splits to fit(). My validation set is not taken from the same distribution as my train set, so I would like to pass it in independently. The use case is seeing reported metrics on the validation set after each epoch.

**Comment by mikerossgithub** [comment]
oops, its in the options as `val_set`
